### PR TITLE
Fix: #7357 Add Release Notes for iOS Release 1.50

### LINKED ISSUE #7357: Release Notes for iOS Release 1.50
### Web3

- Added support for opening ipns:// scheme URL from other apps. ([#7227](https://github.com/brave/brave-ios/issues/7227))
- Added domain resolution support for Unstoppable Domains. ([#7089](https://github.com/brave/brave-ios/issues/7089))
- Added support for IPFS Navigation in omnibox. ([#7013](https://github.com/brave/brave-ios/issues/7013))
- Added checksum validation of addresses on the Send screen in Brave Wallet. ([#7136](https://github.com/brave/brave-ios/issues/7136))

### General

- Added Brave News Widgets feature for home screen and lock screen. ([#7168](https://github.com/brave/brave-ios/issues/7168))
- Added capability to add custom filter lists. ([#6841](https://github.com/brave/brave-ios/issues/6841))
- Added support for scriptlets using uBlock Origin format. ([#7194](https://github.com/brave/brave-ios/issues/7194))
- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. ([#6991](https://github.com/brave/brave-ios/issues/6991))
- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. ([#7213](https://github.com/brave/brave-ios/issues/7213))
- Enhanced Brave VPN to use WireGuard protocol by default. ([#6646](https://github.com/brave/brave-ios/issues/6646))
- Improved caching for VPN SKU credentials. ([#7196](https://github.com/brave/brave-ios/issues/7196))
- Improved load time for adblock filter lists. ([#7147](https://github.com/brave/brave-ios/issues/7147))
- Improved storage management when re-launching the browser. ([#7114](https://github.com/brave/brave-ios/issues/7114))
- Fixed crash with adblock engine/cosmetic filters. ([#6254](https://github.com/brave/brave-ios/issues/6254))
- Fixed crash for Playlist when entering picture-in-picture mode. ([#7347](https://github.com/brave/brave-ios/issues/7347))
- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. ([#7345](https://github.com/brave/brave-ios/issues/7345))
- Fixed flickering or low resolution favicon in Playlist and on New Tab page. ([#7271](https://github.com/brave/brave-ios/issues/7271))
- Fixed images/thumbnails not displaying on sites such as Mastodon and Airbnb with Shields up. ([#7215](https://github.com/brave/brave-ios/issues/7215))
- Fixed padding for items under "Saved Logins" section in Password Manager when syncing. ([#7197](https://github.com/brave/brave-ios/issues/7197))
- Fixed white gradient appearing on the URL bar in private mode. ([#7132](https://github.com/brave/brave-ios/issues/7132))
- Fixed favicon size on lock screen widget. ([#7124](https://github.com/brave/brave-ios/issues/7124))

### PR DESCRIPTION
**Pull request by Uni-verse**
## Summary of Changes

This pull request fixes #7357 

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New or updated UI has been tested across:
  - [ ] Light & dark mode
  - [ ] Different size classes (iPhone, landscape, iPad)
  - [ ] Different dynamic type sizes

## Test Plan:

N/A

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## [1.50](https://github.com/brave/brave-ios/releases/tag/v1.50)
+
+### Web3
+
+- Added support for opening ipns:// scheme URL from other apps. (#[7227](https://github.com/brave/brave-ios/issues/7227))
+- Added support for Unstoppable Domains Navigation. (#[7089](https://github.com/brave/brave-ios/issues/7089))
```
Comment (srirambv) [inline CHANGELOG.md]: Added domain resolution support for Unstoppable Domains.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+- Added support for IPFS Navigation in omnibox. (#[7013](https://github.com/brave/brave-ios/issues/7013))
+- Added validation for Brave Wallet Send to disable the "Send" button when address is missing checksum.  (#[7136](https://github.com/brave/brave-ios/issues/7136))
```
Comment (srirambv) [inline CHANGELOG.md]: can be simplified to "Added checksum validation of addresses on the Send screen in Brave Wallet"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+- Fixed favicons displaying very small on lock screen widget. (#[7124](https://github.com/brave/brave-ios/issues/7124))
```
Comment (srirambv) [inline CHANGELOG.md]: Not a web3 issue. Should go under general

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+- Fixed connecting wallet issue on poap.fun dApp. (#[5727](https://github.com/brave/brave-ios/issues/5727))
```
Comment (srirambv) [inline CHANGELOG.md]: Changed label of this one. The actual issue is going to be fixed on 1.51 so please remove this one from the list.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. (#[7213](https://github.com/brave/brave-ios/issues/7213))
+- Fixed crash with adblock engine/cosmetic filters. (#[6254](https://github.com/brave/brave-ios/issues/6254))
+- Fixed crash for Playlist when entering PIP mode. (#[7347](https://github.com/brave/brave-ios/issues/7347))
+- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. (#[7345](https://github.com/brave/brave-ios/issues/7345))
+- Fixed flickering or low res favicon in Playlist and on New Tab Page. (#[7271](https://github.com/brave/brave-ios/issues/7271))
+- Fixed images/thumbnails not displaying on sites such as Mastodon and airbnb with Shields up. (#[7215](https://github.com/brave/brave-ios/issues/7215))
+- Fixed/Adjusted padding for items under "Saved Logins" section in Password Manager when syncing. (#[7197](https://github.com/brave/brave-ios/issues/7197))
+- Fixed white gradient appearing on the URL bar in private mode. (#[7132

](https://github.com/brave/brave-ios/issues/7132))
```
Comment (srirambv) [inline CHANGELOG.md]: image

Has some weird whitespace. probably copy paste

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+- Improved caching for VPN SKU credentials. (#[7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. (#[7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved app storage management by cleaning up the tmp folder when re-launching app. (#[7114](https://github.com/brave/brave-ios/issues/7114))
+- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. (#[7213](https://github.com/brave/brave-ios/issues/7213))
+- Fixed crash with adblock engine/cosmetic filters. (#[6254](https://github.com/brave/brave-ios/issues/6254))
+- Fixed crash for Playlist when entering PIP mode. (#[7347](https://github.com/brave/brave-ios/issues/7347))
+- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. (#[7345](https://github.com/brave/brave-ios/issues/7345))
+- Fixed flickering or low res favicon in Playlist and on New Tab Page. (#[7271](https://github.com/brave/brave-ios/issues/7271))
+- Fixed images/thumbnails not displaying on sites such as Mastodon and airbnb with Shields up. (#[7215](https://github.com/brave/brave-ios/issues/7215))
```
Comment (srirambv) [inline CHANGELOG.md]: airbnb -> Airbnb

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,35 @@
...
+- Enhanced ad-blocking to support new uBlock Origin scriptlets format. (#[7194](https://github.com/brave/brave-ios/issues/7194))
+- Improved caching for VPN SKU credentials. (#[7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. (#[7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved app storage management by cleaning up the tmp folder when re-launching app. (#[7114](https://github.com/brave/brave-ios/issues/7114))
+- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. (#[7213](https://github.com/brave/brave-ios/issues/7213))
+- Fixed crash with adblock engine/cosmetic filters. (#[6254](https://github.com/brave/brave-ios/issues/6254))
+- Fixed crash for Playlist when entering PIP mode. (#[7347](https://github.com/brave/brave-ios/issues/7347))
+- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. (#[7345](https://github.com/brave/brave-ios/issues/7345))
+- Fixed flickering or low res favicon in Playlist and on New Tab Page. (#[7271](https://github.com/brave/brave-ios/issues/7271))
```
Comment (srirambv) [inline CHANGELOG.md]: res -> resolution
on New Tab Page -> on the New Tab page

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,35 @@
...
+
+- Added Brave News Widgets feature for home screen and lock screen. (#[7168](https://github.com/brave/brave-ios/issues/7168))
+- Added capability to add custom filter lists. (#[6841](https://github.com/brave/brave-ios/issues/6841))
+- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. (#[6991](https://github.com/brave/brave-ios/issues/6991))
+- Enhanced Brave VPN to use WireGuard protocol by default. (#[6646](https://github.com/brave/brave-ios/issues/6646))
+- Enhanced ad-blocking to support new uBlock Origin scriptlets format. (#[7194](https://github.com/brave/brave-ios/issues/7194))
+- Improved caching for VPN SKU credentials. (#[7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. (#[7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved app storage management by cleaning up the tmp folder when re-launching app. (#[7114](https://github.com/brave/brave-ios/issues/7114))
+- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. (#[7213](https://github.com/brave/brave-ios/issues/7213))
```
Comment (srirambv) [inline CHANGELOG.md]: Maybe move this one after the added section

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,35 @@
...
+### General
+
+- Added Brave News Widgets feature for home screen and lock screen. (#[7168](https://github.com/brave/brave-ios/issues/7168))
+- Added capability to add custom filter lists. (#[6841](https://github.com/brave/brave-ios/issues/6841))
+- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. (#[6991](https://github.com/brave/brave-ios/issues/6991))
+- Enhanced Brave VPN to use WireGuard protocol by default. (#[6646](https://github.com/brave/brave-ios/issues/6646))
+- Enhanced ad-blocking to support new uBlock Origin scriptlets format. (#[7194](https://github.com/brave/brave-ios/issues/7194))
+- Improved caching for VPN SKU credentials. (#[7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. (#[7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved app storage management by cleaning up the tmp folder when re-launching app. (#[7114](https://github.com/brave/brave-ios/issues/7114))
```
Comment (srirambv) [inline CHANGELOG.md]: Maybe change to "Improved storage management when re-launching the browser"

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,35 @@
...
+- Fixed favicons displaying very small on lock screen widget. (#[7124](https://github.com/brave/brave-ios/issues/7124))
+- Fixed connecting wallet issue on poap.fun dApp. (#[5727](https://github.com/brave/brave-ios/issues/5727))
+
+### General
+
+- Added Brave News Widgets feature for home screen and lock screen. (#[7168](https://github.com/brave/brave-ios/issues/7168))
+- Added capability to add custom filter lists. (#[6841](https://github.com/brave/brave-ios/issues/6841))
+- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. (#[6991](https://github.com/brave/brave-ios/issues/6991))
+- Enhanced Brave VPN to use WireGuard protocol by default. (#[6646](https://github.com/brave/brave-ios/issues/6646))
+- Enhanced ad-blocking to support new uBlock Origin scriptlets format. (#[7194](https://github.com/brave/brave-ios/issues/7194))
```
Comment (srirambv) [inline CHANGELOG.md]: maybe change to "Added support for scriptlets using uBlock Origin format" ?

**Review by srirambv — CHANGES_REQUESTED** [review]
All the issue numbers are in `(#[Issue number](` format. please change it to `(#Issue number](` format

File: CHANGELOG.md (hunk 3 above)
Comment (Uni-verse) [reply CHANGELOG.md]: Thanks, don't know how i missed that.

**Comment by Uni-verse** [comment]
> All the issue numbers are in `(#[Issue number](` format. please change it to `(#Issue number](` format

Fixed.

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,34 @@
...
+- Improved caching for VPN SKU credentials. ([#7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. ([#7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved storage management when re-launching the browser. ([#7114](https://github.com/brave/brave-ios/issues/7114))
+- Fixed crash with adblock engine/cosmetic filters. ([#6254](https://github.com/brave/brave-ios/issues/6254))
+- Fixed crash for Playlist when entering PIP mode. ([#7347](https://github.com/brave/brave-ios/issues/7347))
+- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. ([#7345](https://github.com/brave/brave-ios/issues/7345))
+- Fixed flickering or low resolution favicon in Playlist and on New Tab page. ([#7271](https://github.com/brave/brave-ios/issues/7271))
+- Fixed images/thumbnails not displaying on sites such as Mastodon and Airbnb with Shields up. ([#7215](https://github.com/brave/brave-ios/issues/7215))
+- Fixed/Adjusted padding for items under "Saved Logins" section in Password Manager when syncing. ([#7197](https://github.com/brave/brave-ios/issues/7197))
```
Comment (srirambv) [inline CHANGELOG.md]: nit: can just be fixed.

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,34 @@
...
+- Added capability to add custom filter lists. ([#6841](https://github.com/brave/brave-ios/issues/6841))
+- Added support for scriptlets using uBlock Origin format. ([#7194](https://github.com/brave/brave-ios/issues/7194))
+- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. ([#6991](https://github.com/brave/brave-ios/issues/6991))
+- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. ([#7213](https://github.com/brave/brave-ios/issues/7213))
+- Enhanced Brave VPN to use WireGuard protocol by default. ([#6646](https://github.com/brave/brave-ios/issues/6646))
+- Improved caching for VPN SKU credentials. ([#7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. ([#7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved storage management when re-launching the browser. ([#7114](https://github.com/brave/brave-ios/issues/7114))
+- Fixed crash with adblock engine/cosmetic filters. ([#6254](https://github.com/brave/brave-ios/issues/6254))
+- Fixed crash for Playlist when entering PIP mode. ([#7347](https://github.com/brave/brave-ios/issues/7347))
```
Comment (srirambv) [inline CHANGELOG.md]: nit: PiP instead of PIP. or better to say `picture-in-picture`

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. ([#7345](https://github.com/brave/brave-ios/issues/7345))
+- Fixed flickering or low resolution favicon in Playlist and on New Tab page. ([#7271](https://github.com/brave/brave-ios/issues/7271))
+- Fixed images/thumbnails not displaying on sites such as Mastodon and Airbnb with Shields up. ([#7215](https://github.com/brave/brave-ios/issues/7215))
+- Fixed/Adjusted padding for items under "Saved Logins" section in Password Manager when syncing. ([#7197](https://github.com/brave/brave-ios/issues/7197))
+- Fixed white gradient appearing on the URL bar in private mode. ([#7132](https://github.com/brave/brave-ios/issues/7132))
+- Fixed favicons displaying very small on lock screen widget. ([#7124](https://github.com/brave/brave-ios/issues/7124))
```
Comment (srirambv) [inline CHANGELOG.md]: maybe change to "Fixed favicon size on lock screen widget" ?

**Review by srirambv — CHANGES_REQUESTED** [review]
few nits but other than that looks good to me. good job 👍

**Review by srirambv — APPROVED** [review]
++

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [1.50](https://github.com/brave/brave-ios/releases/tag/v1.50)
+
+### Web3
+
+- Added support for opening ipns:// scheme URL from other apps. ([#7227](https://github.com/brave/brave-ios/issues/7227))
+- Added domain resolution support for Unstoppable Domains. ([#7089](https://github.com/brave/brave-ios/issues/7089))
+- Added support for IPFS Navigation in omnibox. ([#7013](https://github.com/brave/brave-ios/issues/7013))
+- Added checksum validation of addresses on the Send screen in Brave Wallet. ([#7136](https://github.com/brave/brave-ios/issues/7136))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: we usually put specific areas of UI in quote. For example:

```
- Added checksum validation of addresses on the "Send" screen in Brave Wallet. ([#7136](https://github.com/brave/brave-ios/issues/7136))
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,34 @@
...
+
+### General
+
+- Added Brave News Widgets feature for home screen and lock screen. ([#7168](https://github.com/brave/brave-ios/issues/7168))
+- Added capability to add custom filter lists. ([#6841](https://github.com/brave/brave-ios/issues/6841))
+- Added support for scriptlets using uBlock Origin format. ([#7194](https://github.com/brave/brave-ios/issues/7194))
+- Added “Never Saved” section in Logins & Passwords Page when syncing passwords. ([#6991](https://github.com/brave/brave-ios/issues/6991))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: I think we're using different quotes here? `“Never Saved”` vs `"Never Saved"`. Can barely tell the difference though so not the biggest deal.

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,5 +1,34 @@
...
+- Blocked Playlist from appearing in menu and URL bar when using Brave Talk. ([#7213](https://github.com/brave/brave-ios/issues/7213))
+- Enhanced Brave VPN to use WireGuard protocol by default. ([#6646](https://github.com/brave/brave-ios/issues/6646))
+- Improved caching for VPN SKU credentials. ([#7196](https://github.com/brave/brave-ios/issues/7196))
+- Improved load time for adblock filter lists. ([#7147](https://github.com/brave/brave-ios/issues/7147))
+- Improved storage management when re-launching the browser. ([#7114](https://github.com/brave/brave-ios/issues/7114))
+- Fixed crash with adblock engine/cosmetic filters. ([#6254](https://github.com/brave/brave-ios/issues/6254))
+- Fixed crash for Playlist when entering picture-in-picture mode. ([#7347](https://github.com/brave/brave-ios/issues/7347))
+- Fixed issue causing Playlist to play a short chime instead of the actual video that was added. ([#7345](https://github.com/brave/brave-ios/issues/7345))
+- Fixed flickering or low resolution favicon in Playlist and on New Tab page. ([#7271](https://github.com/brave/brave-ios/issues/7271))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: we usually use `New Tap Page` when referring to NTP.

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,5 +1,34 @@
...
+- Fixed images/thumbnails not displaying on sites such as Mastodon and Airbnb with Shields up. ([#7215](https://github.com/brave/brave-ios/issues/7215))
```
Comment (kjozwiak) [inline CHANGELOG.md]: I think something like the following sounds better than using "Shields up":

```
- Fixed images/thumbnails not displaying on sites such as Mastodon and Airbnb when Brave Shields is enabled. ([#7215](https://github.com/brave/brave-ios/issues/7215))
```

**Review by kjozwiak — DISMISSED** [review]
A few small nit but looking good! Go ahead and merge once addressed. There's no need to request review again.

File: CHANGELOG.md (hunk 15 above)
Comment (Uni-verse) [reply CHANGELOG.md]: Nice find, fixed!

**Pull request merged**